src/scripts/ibmcloud-auth.js

```diff
@@ -111,6 +127,40 @@ if (env.readerUsers) {
 	READER_EMAILS = env.readerUsers.split(',');
 }
 
+function ldapInit() {
+	var promise = new Promise((resolve, reject) => {
+		if (env.ldapServer && env.ldapPort && env.ldapBindUser && env.ldapBindPassword) {
+			var ldapUrl = `${env.ldapProtocol}://${env.ldapServer}:${env.ldapPort}`;
+			POWER_GROUPS = env.ldapPowerGroups.split(';');
```

Comment:
Why semicolon?  The other env uses commas.
HUBOT_IBMCLOUD_POWERUSERS=comma-separated list of power-user emails -- no spaces!
HUBOT_IBMCLOUD_READERUSERS=comma-separated list of reader-user emails -- no spaces!